package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "start": "cross-env NODE_PATH=. node server.js",
     "dev": "yarn build && cross-env NODE_PATH=. node-dev server.js",
+    "dev-mock-api": "concurrently \"yarn dev\" \"dyson test/mock/ 8080\"",
```

Comment:
I think it might be useful to put the dyson command in it's own script in case any one wants to run it standalone.